p/themes/Alternative-Dark/adark.css

```diff
@@ -348,7 +348,7 @@ a.btn {
 .dropdown-menu .item > .as-link {
 	padding: 0 22px;
 	line-height: 2.5;
-	font-size: 0.8rem;
+	font-size: inherit;
```

Comment:
Wouldn't removing the line have the same effect?

Reply:
I thought so, too. But it does not have the same effect.

`font-size: inherit` = 12,8pixel
deleted line = 13.33 pixel

To be honest: I did not understand why it has not the same effect.

Reply:
> I did not understand why it has not the same effect

I have not checked, but this is typically due to the fact that without this line, another rule is applied from another selector.

Reply:
The inspector will say which rule. Anyway, just double checking.